kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/S3SinkConnectorIT.java

```diff
@@ -0,0 +1,288 @@
+      String kafkaTopic = KAFKA_TOPICS.get(i % KAFKA_TOPICS.size());
+      String kafkaKey = "simple-key-" + i;
+      String kafkaValue = "simple-message-" + i;
+      log.debug("Sending message {} with topic {} to Kafka broker {}", kafkaTopic, kafkaValue);
+      connect.kafka().produce(kafkaTopic, kafkaKey, kafkaValue);
+    }
+  }
+
+  private Map<String, String> getConnectorProps() {
+    Map<String, String> props = new HashMap<>();
+    props.put(SinkConnectorConfig.TOPICS_CONFIG, String.join(",", KAFKA_TOPICS));
+    props.put(CONNECTOR_CLASS_CONFIG, "io.confluent.connect.s3.S3SinkConnector");
+    props.put(TASKS_MAX_CONFIG, Integer.toString(TASKS_MAX));
+
+    props.put("s3.region", "ap-south-1");
+    props.put("s3.part.size", "5242880");
+    props.put("s3.bucket.name", S3_BUCKET);
+    props.put("flush.size", Integer.toString(FLUSH_SIZE));
+    props.put("storage.class","io.confluent.connect.s3.storage.S3Storage");
+    props.put("partitioner.class", "io.confluent.connect.storage.partitioner.DefaultPartitioner");
+
+    // converters
+    props.put(KEY_CONVERTER_CLASS_CONFIG, StringConverter.class.getName());
+    props.put(VALUE_CONVERTER_CLASS_CONFIG, StringConverter.class.getName());
+
+    props.put("format.class",JSON_FORMAT_CLASS);
+    // license properties
+    return props;
+  }
+
+  private void deleteBucket(String bucketName) {
+    emptyBucket(bucketName);
+    // After all objects are deleted, delete the bucket.
+    s3RootClient.deleteBucket(bucketName);
+  }
+
+  private void emptyBucket(String bucketName) {
+    // delete all objects to empty bucket
+    ObjectListing objectListing = s3RootClient.listObjects(bucketName);
+    while (true) {
+      Iterator<S3ObjectSummary> objIter = objectListing.getObjectSummaries().iterator();
+      while (objIter.hasNext()) {
+        s3RootClient.deleteObject(bucketName, objIter.next().getKey());
+      }
+
+      if (objectListing.isTruncated()) {
+        objectListing = s3RootClient.listNextBatchOfObjects(objectListing);
+      } else {
+        break;
+      }
+    }
+    // delete versioned objects
+    VersionListing versionList = s3RootClient.listVersions(new ListVersionsRequest().withBucketName(bucketName));
+    while (true) {
+      Iterator<S3VersionSummary> versionIter = versionList.getVersionSummaries().iterator();
+      while (versionIter.hasNext()) {
+        S3VersionSummary vs = versionIter.next();
+        s3RootClient.deleteVersion(bucketName, vs.getKey(), vs.getVersionId());
+      }
+
+      if (versionList.isTruncated()) {
+        versionList = s3RootClient.listNextBatchOfVersions(versionList);
+      } else {
+        break;
+      }
+    }
+  }
+
+  private void createS3Bucket(String bucketName) {
+    if (!s3RootClient.doesBucketExistV2(bucketName)) {
+      s3RootClient.createBucket(new CreateBucketRequest(bucketName));
+    }
+  }
+
+  public static void setupSquidProxy() {
+    squid = new SquidProxy("connect-squid-local:1.0.0", "NONE");
```

Comment:
The image name is fine when you are running on local. But while committing the code keep it as it was earlier (with confluent docker url).